src/crosschain-user.ts

```diff
@@ -10,92 +10,101 @@ import { getMetamaskSigner } from './solidity-helpers'
 
 const log = debug('crosschain-user')
 
+export interface CrossChainUserParams {
+  // Dappchain endpoint settings
+  dappchainEndpoint: string
+  chainId: string
+
+  // Keys
+  dappchainPrivateKey?: string
+  ethereumPrivateKey?: string
+
+  // Ethereum endpoint settings
+  web3?: Web3
+  ethEndpoint?: string
+  wallet?: ethers.Signer
+}
+
+export interface NewCrossChainUserParams {
```

Comment:
Looks like this is supposed to be the constructor params interface, so best to call it `CrossChainUserConstructorParams`.